app/assets/javascripts/admin/templates/web-hooks-show.hbs

```diff
@@ -0,0 +1,78 @@
+{{#link-to 'adminWebHooks' class="go-back"}}
+  {{fa-icon 'arrow-left'}}
+  {{i18n 'admin.web_hooks.go_back'}}
+{{/link-to}}
+
+<div class='web-hook-container'>
+  <p>{{i18n 'admin.web_hooks.detailed_instruction'}}</p>
+  <form class='web-hook form-horizontal'>
+    <div>
+      <label for='payload-url'>{{i18n 'admin.web_hooks.payload_url'}}</label>
+      {{text-field name="payload-url" value=model.payload_url placeholderKey="admin.web_hooks.payload_url_placeholder"}}
+    </div>
+
+    <div>
+      <label for='content-type'>{{i18n 'admin.web_hooks.content_type'}}</label>
+      {{combo-box content=model.contentTypes
+                  name="content-type"
+                  nameProperty="name"
+                  valueAttribute="id"
+                  value=model.content_type}}
+    </div>
+
+    <div>
+      <label for='secret'>{{i18n 'admin.web_hooks.secret'}}</label>
+      {{text-field name="secret" value=model.secret placeholderKey="admin.web_hooks.secret_placeholder"}}
+    </div>
+
+    <div class="cbox10">
+      <label>{{i18n 'admin.web_hooks.event_chooser'}}</label>
+      <div>
+        {{radio-button class="subscription-choice" name="subscription-choice" value=false selection=model.wildcardEvent}}
+        {{i18n 'admin.web_hooks.individual_event'}}
+      </div>
+      {{#unless model.existWildcardEvent}}
+        <div class="event-selector">
+          {{#each model.EVENTS as |type|}}
+            {{admin-web-hook-event-chooser type=type model=model.web_hook_event_types}}
+          {{/each}}
+        </div>
+      {{/unless}}
+      <div>
+        {{radio-button class="subscription-choice" name="subscription-choice" value=true selection=model.wildcardEvent}}
+        {{i18n 'admin.web_hooks.wildcard_event'}}
+      </div>
+    </div>
+
+    <div class='filters'>
+      <div>
+        <label><span class="icon fa fa-circle tracking"></span> {{i18n 'admin.web_hooks.category_filters'}}</label>
+        {{category-group categories=model.categoryFilters blacklist=model.categoryFilters}}
+        <div class="instructions">{{i18n 'admin.web_hooks.category_filters_instructions'}}</div>
+      </div>
+    </div>
+
+    <div>
+      {{input type="checkbox" name="verify_certificate" checked=model.verify_certificate}} {{i18n 'admin.web_hooks.verify_certificate'}}
+    </div>
+    <div>
+      <div>
+        {{input type="checkbox" name="active" checked=model.active}} {{i18n 'admin.web_hooks.active'}}
+      </div>
+      {{#if model.active}}
+        <div class="instructions">{{i18n 'admin.web_hooks.active_notice'}}</div>
+      {{/if}}
+    </div>
+  </form>
+
+  <div class='controls'>
+    <button {{action 'save'}} disabled={{model.isSaving}} class='btn btn-default'>{{saveButtonText}}</button>
+    {{#unless model.isNew}}
+      {{d-button class="btn-danger" label="admin.web_hooks.destroy" action="destroy"}}
+    {{/unless}}
+    <span class='saving'>{{savingStatus}}</span>
+  </div>
+
+  <table class='web-hook-events'>
```

Comment:
Mind a placeholder here? I will follow this with a table of historical webhook events.